src/app/shared/table/table.component.ts

```diff
@@ -137,6 +138,22 @@ export class TableComponent implements OnInit, OnDestroy, AfterViewInit {
     return '';
   }
 
+  getAdvancedLink(index: number, tableColumn: TableColumn): string {
+    let linkTmp = tableColumn.linkPattern || '';
```

Comment:
please create a method getAdvancedData that will be used for the advancedDisplay and also for the advancedLink. Pass another parameter "type", so you would know what pattern and params you should use.

getAdvancedData(index, tableColumn, type) ..... type = "display" or "link"

So we don't repeat ourselfs, cause those 2 methods basicly do the same thing

Reply:
Done, thx!